automation/test.sh

```diff
@@ -176,6 +176,13 @@ set -e
 echo "Nodes are ready:"
 kubectl get nodes
 
+# Create .bazelrc to use remote cache
```

Comment:
The `test.sh` script still runs `make pull-cache`. Bazel can not make use of it. I think we should remove it, because it doubles the amount of things to download.